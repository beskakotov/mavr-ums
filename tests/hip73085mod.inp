Object: HIP73085
RA:     14.9369
Dec:    17.7482
P       34.6611 0.0829768
T       2015.25 0.186604
e       0.407261 1.24849
a       234.504 1.83228
W       27.0169 0.568115
w       192.999 1.51858
i       111.608 0.595671
*K1 
*K2 
*V0 
1991.25 45.9642 240 1000 10.7041 I1HIP1997a_BP
2001.27 29.2052 277 10 13.896 I1Bali06
2007.33 8.62996 212 1000 338.681 I1Hor10
2007.331 8.72997 198 10 338.553 I1Hor10
2008.47 4.23465 180 10 327.523 I1Hor12
2002.33 27.0095 270 10 14.3464 I1Bali13
2002.331 27.2095 269 10 14.5464 I1Bali13
2002.332 27.4095 275 10 14.7467 I1Bali13
2014.22 240.458 82.1 10 28.1136 I1Hor15
2014.221 240.758 82.6 10 28.4137 I1Hor15
2014.46 234.259 85.5 10 24.2624 I1Hor15
2014.461 235.359 81.9 10 25.3625 I1Hor15
2009.41 358.364 156.49 10 47.2645 I1Mitr_NP
2012.42 307.038 74.42 10 73.4841 I1Mitr_NP
2013.32 275.114 67.1 10 53.3946 I1Mitr_NP
2014.28 241.324 84.03 10 29.5607 I1Mitr_NP
2015.44 212.957 115.13 10 12.8067 I1Mitr_NP
2016.13 204.516 122.93 10 12.752 I1Mitr_NP
2016.4 202.032 125.35 10 14.1093 I1Mitr_NP
2018.08 183.882 134.12 10 40.5424 I1Mitr_NP
2019.27 172.074 134.4 10 78.5292 I1Mitr_NP
2019.271 171.926 134.06 10 78.3815 I1Mitr_NP
2019.42 170.122 132.98 10 81.3264 I1Mitr_NP
