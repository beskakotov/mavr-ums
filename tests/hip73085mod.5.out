Object: HIP73085
RA:     14.9369
Dec:    17.7482
P	43.6576 0.569297
T0	2014.66 0.0601296
e	0.47536 0.00390356
a	216.425 1.60762
W	22.5957 0.554877
w	136.968 1.31529
i	123.969 0.784582
*K1
*K2
*V0
CHISQ	Sig_V1	Sig_V2	Sig_T	Sig_Ro
3.76237	0	0	2.44376	4.11318
1991.25 45.9642 1000 -8.76456 -9.75789 45.9642 I1HIP1997a
2001.27 29.2052 10 -0.501329 -2.17941 29.2052 I1Bali06
2007.33 8.62996 1000 -2.0175 9.14844 8.62996 I1Hor10
2007.33 8.72997 10 -1.91287 -4.83048 8.72997 I1Hor10
2008.47 4.23465 10 -0.393629 3.222 4.23465 I1Hor12
2002.33 27.0095 10 -0.0499686 -2.39944 27.0095 I1Bali13
2002.33 27.2095 10 0.152595 -3.39189 27.2095 I1Bali13
2002.33 27.4095 10 0.355158 2.61566 27.4095 I1Bali13
2014.22 240.458 10 -0.140501 -2.208 240.458 I1Hor15
2014.22 240.758 10 0.186209 -1.72842 240.758 I1Hor15
2014.46 234.259 10 -0.286536 -3.85455 234.259 I1Hor15
2014.46 235.359 10 0.837243 -7.47604 235.359 I1Hor15
2009.41 358.364 10 0.351552 4.18242 358.364 I1Mitr_NP
2012.42 307.038 10 2.91135 0.417343 307.038 I1Mitr_NP
2013.32 275.114 10 5.11804 -3.69555 275.114 I1Mitr_NP
2014.28 241.324 10 2.30542 -1.5137 241.324 I1Mitr_NP
2015.44 212.957 10 -2.7068 5.32548 212.957 I1Mitr_NP
2016.13 204.516 10 -1.33693 1.91136 204.516 I1Mitr_NP
2016.4 202.032 10 -0.417118 0.955091 202.032 I1Mitr_NP
2018.08 183.882 10 0.155107 0.142167 183.882 I1Mitr_NP
2019.27 172.074 10 0.962105 1.2618 172.074 I1Mitr_NP
2019.27 171.926 10 0.824821 0.923758 171.926 I1Mitr_NP
2019.42 170.122 10 0.621088 0.146443 170.122 I1Mitr_NP
